test/integration/prerender/test/index.test.js

```diff
@@ -428,6 +428,31 @@ const runTests = (dev = false, looseMode = false) => {
     //   )
     // })
 
+    it('should not show error from url prop being returned', async () => {
```

Comment:
This test should actually check the `url` value gets through unchanged, not just a lack of error.

Reply:
Updated